lib/shelley/src/Cardano/Wallet/Shelley/Network.hs

```diff
@@ -790,6 +792,8 @@ instance ToText NetworkLayerLog where
             [ "Destroying cursor connection at"
             , T.pack (show threadId)
             ]
+        MsgWillQueryRewardsForStake c ->
+            "Will query non-myopic rewards using the stake " <> pretty c
```

Comment:
Is the balance private and not to be logged?

Reply:
Nope, fine. Balances are already logged as part of the wallet. Current logs aren't much private.